pywebostv/controls.py

```diff
@@ -136,7 +140,10 @@ def callback_wrapper(res):
 
     def subscribe(self, name, cmd_info):
         def request_func(callback):
-            response_valid = cmd_info.get("validation", lambda p: (True, None))
+            response_valid = cmd_info.get(
```

Comment:
Instead of modifying the retrieval of validation function here, let's keep as-is, and pass `subscribe_validation` to get_audio_output` instead?

Reply:
Unfortunately, for this specific method (get_audio_output), somehow the direct call and subscribed called differ in response from the server, but other works just fine as it is, so I tried to keep compatibility while allowing separate validators

Reply:
Got it, in that case -- I think let's have the implementation to be very explicit -- `subscribe(..)` will always look for `subscribe_validation` field. In other words, let's not fallback to regular `validation`. We might have to update all other subscribe-able APIs. Curious to know what you think..

Reply:
Yeah, I agree, less complex that way, will do

Reply:
It's what you expect `subscription_validation` to be something like:
```python
def subscription_validation(payload):
    for key in {"subscribed", "returnValue"}:
        status, error_text = standard_validation(payload, key)
        if status:
            break
    return status, error_text
```
?